azure/flatcar-linux/kubernetes/workers/workers.tf

```diff
@@ -42,13 +42,14 @@ resource "azurerm_virtual_machine_scale_set" "workers" {
     custom_data          = "${data.ct_config.worker-ignition.rendered}"
   }
 
-  # Azure mandates setting an ssh_key, even though Ignition custom_data handles it too
+  # Azure mandates setting an ssh_key, provide just a single key as the
+  # others are handled with Ignition custom_data
```

Comment:
Ditto dot here.

Reply:
Still missing?

Reply:
I haven't pushed yet because was trying do so something about the ssh_keys :P.